src/main/java/net/datafaker/service/FakeValuesService.java

```diff
@@ -720,7 +720,7 @@ private Object resolveExpression(String directive, String[] args, Object current
         // did first but FIRST we change the Object reference Class.method_name with a yml style internal reference ->
         // class.method_name (lowercase)
         if (dotDirective) {
-            supplier = () -> safeFetch(directive, null);
+            supplier = () -> safeFetch(directive.toLowerCase(), null);
```

Comment:
This was needed to address failures within certain provider configurations that used title-case for the classname, like Address.street_title instead of address.street_title. The config classes could be updated or we could simply make it case-insensitive, as I have here.